scenarios/README.md

```diff
@@ -401,6 +401,41 @@ The scenario named `custom` can be used to pass any custom filter variable like
 crank --config https://github.com/aspnet/benchmarks/blob/main/scenarios/dotnet.benchmarks.yml?raw=true --scenario custom --profile aspnet-perf-win --variable filter=*LinqBenchmarks*
 ```
 
+## HTTP client benchmarks
+
+These scenarios are running various HTTP client benchmarks.
+
+### Sample
+
+```
+crank --config https://raw.githubusercontent.com/CarnaViire/Benchmarks/http3-benchmarks/scenarios/httpclient.benchmarks.yml --scenario httpclient-kestrel-get --profile aspnet-perf-lin
+```
+*TODO: update to `https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/httpclient.benchmarks.yml`*
+
+### Available scenarios
+
+- `httpclient-kestrel-get`: Execute GET requests to the server. Client is HttpClient, server is Kestrel (minimal API).
+
+#### Arguments
+
+- HTTP Version to use:
+  - `--variable httpVersion="1.1"` (default)
+  - `--variable httpVersion="2.0"`
+  - `--variable httpVersion="3.0"`
+- Number of clients:
+  - `--variable numberOfClients=<N>` (default: `1`)
+- Number of concurrect requests per one client:
+  - `--variable concurrencyPerClient=<N>` (default: `12`)
+- Max number of HTTP/1.1 connections per server:
+  - `--variable http11MaxConnectionsPerServer=<N>` (default: `1`)
```

Comment:
What happens with all the defaults: httpVersion=1.1, concurrencyPerClient=12, http11MaxConnectionsPerServer=1

One HttpClient instance, 12 concurrent tasks can use it, but a single connection allowed? To me it looks like `http11MaxConnectionsPerServer` should not be set and be driven by `concurrencyPerClient` indirectly. But might be my limited knowledge of HttpClient.

Reply:
Concurrent tasks end up competing for a single HTTP/1 connection, yes. But that would be an honest comparison with HTTP/2 and 3, that also have 1 connection, but use it more efficiently via multiplexing different requests on a single connection.

Reply:
Infinite / concurrencyPerClient may be a better default. While it's great that we have an option to test the contended scenario, I think we will for the most part be interested in the actual throughput of the system and not a single connection.

Reply:
I agree, the basic scenario should be infinite connections for all HTTP versions.

Reply:
Ok, since there's no way to set the number of connections strictly to concurrencyPerClient for all HTTP versions, then I see `http11MaxConnectionsPerServer=int.MaxValue` and `http20EnableMultipleConnections=true` as defaults will be closest to what you want. We would still have only one connection for HTTP/3 though.